tests/spread/core24/grammar/snap/snapcraft.yaml

```diff
@@ -24,6 +26,24 @@ build-packages:
 parts:
   hello-world:
     plugin: dump
+
+    # grammar-only string
     source:
     - on amd64 to amd64: src/on-amd64-to-amd64
     - on amd64 to arm64: src/on-amd64-to-arm64
+
+    # grammar dictionary
+    organize:
```

Comment:
Is a grammar dict with some non-grammar components valid?

Reply:
I don't think so.  At least this isn't a regression because it's never worked in core20 or core22.

I made a task in craft-application to support it: https://github.com/canonical/craft-application/issues/331